dbc2val/Readme.md

```diff
@@ -124,6 +132,25 @@ Configuration options have the following priority (highest at top).
 3. configuration file
 4. default value
 
+### USE_J1939
+
+This environment variable indicates if J1939 mode shall be used, and shall be `1` to indicate that J1939 mode shall be used.
+All other values will result in that J1939 mode is not used.
+
+
+### VDB_ADDRESS
```

Comment:
Names of env-variables to give address and port seems to be quite inconsistent. In [databroker](https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker#configuration) `KUKSA_DATA_BROKER_ADDR` and `KUKSA_DATA_BROKER_PORT` is used, which differs a bit from what is used here. Do we actually need to have specific env-variables for Databroker and Server, respectively.

I am thinking for clients if we could just use `KUKSA_ADDR` and `KUKSA_PORT` and support them for both server and broker. Then we could also remove `DAPR_GRPC_PORT`, you could use the generic `KUKSA_ADDR` and `KUKSA_PORT` for that use case as well.

Reply:
No real opinion here. Sounds reasonable. Not a big user of env vars, but they can come in handy in K8/3/0S deployments